javaparser-core/src/main/java/com/github/javaparser/ast/nodeTypes/NodeWithMembers.java

```diff
@@ -112,6 +113,21 @@ default FieldDeclaration addField(Type type, String name, Modifier... modifiers)
     }
 
     /**
+     * Add a field to this

```

Comment:
Nitpick: I think we need a dot after this

Reply:
It's copied from the other `addField`s, so at least it's consistently missing ;-)

Reply:
Yes it is lol